app/src/main/java/org/sopt/dosopttemplate/feature/home/HomeScreen.kt

```diff
@@ -0,0 +1,101 @@
+package org.sopt.dosopttemplate.feature.home
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import org.sopt.dosopttemplate.MainViewModel
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HomeScreen(
+    modifier: Modifier = Modifier,
+    navController: NavController,
+    viewModel: MainViewModel,
+) {
+    Scaffold { innerPadding ->
+        HomeContent(
+            modifier = modifier.padding(innerPadding),
+            navController = navController,
+            viewModel = viewModel,
+        )
+    }
+}
+
+@Composable
+fun HomeContent(
+    modifier: Modifier,
+    navController: NavController,
+    viewModel: MainViewModel,
+) {
```

Comment:
```suggestion
@Composable
fun HomeContent(
    modifier: Modifier = Modifier,
    navController: NavController,
    viewModel: MainViewModel,
) {
```
+) 이렇게 작성하면 Preview를 그릴 수 없겠죠? 어떻게 하면 HomeContent의 Preview를 그릴 수 있게 만들 수 있을까요?